app/controllers/moab_storage_controller.rb

```diff
@@ -29,7 +29,7 @@ def show
 
   def druids_from_storage_root
     @druids ||= begin
-      @storage_root ||= "#{Moab::Config.storage_roots}/#{Moab::Config.storage_trunk}"
+      @storage_root ||= "#{Moab::Config.storage_roots.first}/#{Moab::Config.storage_trunk}"
```

Comment:
👍 - elegant quick solution to the problem.

Reply:
thanks!